k8s/namespaces/nfdiv/nfdiv-cron-notify-respondent-apply-final-order/nfdiv-cron-notify-respondent-apply-final-order.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-notify-respondent-apply-final-order
+spec:
+  releaseName: nfdiv-cron-notify-respondent-apply-final-order
+  chart:
+    git: git@github.com:hmcts/nfdiv-cron
+    ref: 0.0.8
```

Comment:
up to you but we generally avoid versioning these as people don't normally understand the order of operations or just forget to update these forever.

Pointing to master we've found has been safer

Reply:
There was a reason for that but I haven't got the foggiest as to what it was: https://github.com/hmcts/nfdiv-cron/commit/2e43dc956d2230033c8a821cfd5affd0f64eb6a6

Reply:
fair enough, we've got a lot of crons so can we address that en masse in a separate PR?